src/main/java/com/glencoesoftware/omero/zarr/OmeroAmazonS3ClientFactory.java

```diff
@@ -55,23 +68,36 @@ protected AWSCredentialsProvider getCredentialsProvider(Properties props) {
                     + " Please use either named profiles or instance"
                     + " profile credentials.");
         }
-        boolean anonymous = Boolean.parseBoolean(
-                (String) props.get("s3fs_anonymous"));
+
+        boolean anonymous = false;
```

Comment:
The PR description and comments should be updated to describe the new logic for anonymous detection.
From my reading, an AWS profile named `s3fs_anonymous` needs to be configured with both the access and secret access keys set to `s3fs_anonymous`. 

Is that correct and if so, is there a reference for this strategy and is it compatible with other native tools like AWS CLI?

Reply:
My idea being the introduction of this option is that we could have different profiles in the ``.aws/credentials`` based on endpoint.

Also the current "profile" logic is based on information in url. From our discussion this was a previous approach and not something we should retain, so that part of the code will also need to be reviewed afterwards